components/NavBar/index.tsx

```diff
@@ -2,25 +2,24 @@ import { useCallback, useState } from 'react'
 import Link from 'next/link'
 import dynamic from 'next/dynamic'
 import NavLink, { NavLinkProps } from './NavLink'
+import Image from 'next/image'
+import logo from '../../public/images/logoFoodbankAus.webp'
 
-const Auth = dynamic(
-  () => import('../Auth'),
-  { ssr: false }
-)
+const Auth = dynamic(() => import('../Auth'), { ssr: false })
 
 const links: Array<NavLinkProps> = [
   {
-    page: 'Superhero Foods',
-    route: '/'
+    page: 'SUPERHERO FOODS',
+    route: '/super_foods' //placeholder
   },
   {
-    page: 'Recipe',
-    route: '/'
+    page: 'RECIPE',
+    route: '/recipe' //placeholder
   },
   {
-    page: 'Progress',
-    route: '/'
-  },
+    page: 'PROGRESS',
+    route: '/progress' //placeholder
+  }
```

Comment:
This seems like unnecessary changes which do not achieve anything. The font auto capitalizes the words anyways